agent/config/builder.go

```diff
@@ -32,6 +33,15 @@ import (
 	"golang.org/x/time/rate"
 )
 
+// The following constants are default values for some settings
+// Ensure to update documentation if you modify those values
+const (
+	// DefaultEntryFetchMaxBurst is the default value for cache.entry_fetch_max_burst
+	DefaultEntryFetchMaxBurst = 2
```

Comment:
The rate limiter should be disabled by default imho. Which means that the default should be `rate.Inf`.

Reply:
Done in b3947c3f218b1e616c05d8f4cb0876cf7a543cd6